src/components/CollapsibleWrap/CollapsibleWrap.stories.tsx

```diff
@@ -0,0 +1,60 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Button } from "@components/Button/Button";
+import { Meta, Story } from "@storybook/react";
+import React, { useState } from "react";
+import { CollapsibleWrapProps } from "./types";
+import { CollapsibleWrap as CollapsibleWrapComponent } from "./CollapsibleWrap";
+import { Text } from "@typography/Text";
+import { Heading } from "@typography/Heading";
+import { Card } from "@components/Card";
+
+// eslint-disable-next-line import/no-default-export
+export default {
+    title: "Components/Collapsible Wrap",
+    component: CollapsibleWrapComponent,
+} as Meta<CollapsibleWrapProps>;
+
+export const CollapsibleWrap: Story<CollapsibleWrapProps> = () => {
```

Comment:
Please add the [storybook controls](https://storybook.js.org/docs/react/essentials/controls) so we have props documentation for free and a tool to play around :)

Reply:
Added :)

Reply:
Nice 🙌